packages/react-ui/components/SidePage/SidePage.styles.ts

```diff
@@ -159,42 +160,57 @@ export const styles = memoizeStyle({
 
   close(t: Theme) {
     return css`
-      color: ${t.sidePageCloseButtonColor};
+      ${resetButton()};
       cursor: pointer;
-      position: absolute;
-      right: ${t.sidePageCloseButtonPadding};
-      text-align: center;
-      text-decoration: none;
-      width: ${t.sidePageCloseIconSize};
-      padding: 0 ${t.sidePageCloseButtonClickAreaX};
-      margin: 0 -${t.sidePageCloseButtonClickAreaX};
+      color: ${t.sidePageCloseButtonColor};
+      padding: ${t.sidePageCloseButtonClickAreaX};
```

Comment:
Может переименовать тогда переменную? Будет как в модалке. Я думаю вероятность мала, что кто-то ее использует.

```suggestion
      padding: ${t.sidePageCloseButtonClickArea};
```

Reply:
Ну, думаю, можно.